object_detection/good_scripts/convert_to_yolo.py

```diff
@@ -0,0 +1,14 @@
+import cv2
+
+def convert_to_yolo(img_path, x1, y1, width, height):
+
```

Comment:
Could you do a explanation of what this does 

'''
gets x,y,height of image
'''